src/pages/audioPAge/audioPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
the file name should capitalized `AudioPage.js`
also this is a page you can't implement logic in it
so create another component for audio and just call it here

also provide a screen record showing if this code working or not cuz I try it in my computer and it didn't work